srcpkgs/diffoscope/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'diffoscope'
 pkgname=diffoscope
-version=246
-revision=2
+version=249
```

Comment:
251 is available

Reply:
I'm a bit hesitant to update since a couple tests fail. Although in my personal use cases, it works fine. But Gentoo has marked this incompatible with python 3.12 due to the failed test in their [ebuild](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-util/diffoscope/diffoscope-251.ebuild#n7).

cc @Chocimier

Reply:
Fixed now!